styles/globals.scss

```diff
@@ -14,3 +14,8 @@ body {
 * {
   font-family: $font;
 }
+
+input:focus,
+textarea {
+  outline: none;
+}
```

Comment:
Does it make sense to move this to a more local scope instead of global?

Reply:
Using Search component from EDS instead